ciri/BUILD

```diff
@@ -1,10 +1,26 @@
 package(default_visibility = ["//visibility:public"])
 
+load(":conf.bzl", "CIRI_MAINNET_VARIABLES")
+load(":conf.bzl", "CIRI_TESTNET_VARIABLES")
+
+config_setting(
+    name = "mainnet",
+    values = {"define": "network=mainnet"},
+)
+
+config_setting(
+    name = "testnet",
+    values = {"define": "network=testnet"},
+)
+
 cc_binary(
     name = "ciri",
     srcs = ["main.c"],
     copts = ["-DLOGGER_ENABLE"],
-    data = ["ciri.db"],
+    data = [
```

Comment:
Please use a `select({":testnet": ["db/ciri-testnet.db"], ":mainnet": ["db/ciri-mainnet.db"]})` here, no need to package both simultaneously.